server/routes/transport.js

```diff
@@ -72,10 +72,12 @@ function authenticateToken(req, res, next) {
   const authCookie = req.cookies.AccessToken;
   const checkedToken = service.tokenCheck(authCookie);
   if (checkedToken === '401') {
-    return res.sendStatus(401);
-  } if (checkedToken === '403') {
+    res.sendStatus(401);
+  }
+  if (checkedToken === '403') {
```

Comment:
if the return is being removed then the `else` parts need to be added back in or else each of these if statements can be ran technically